src/packages/excalidraw/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Please add the latest change on the top under the correct section.
 
 ### Features
 
+- Changes to i18n [#2644](https://github.com/excalidraw/excalidraw/pull/2644):
+  - BREAKING: removed language picker from UI. It is now host app's responsibility to implement language picker if desirable, using the newly added [`renderFooter`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#renderFooter) prop. The reasoning is that host apps will the i18n be controlled by the app itself, not by the nested Excalidraw component.
+  - Added [`langCode`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#langCode) prop to control language.
+  - Added [`onLangChange`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#onLangChange) callback.
```

Comment:
Is this needed to be mentioned in changelog? As it's already mentioned in the PR description we can avoid mentioning it in the changelog.

Reply:
For a start, we should definitely note what is a breaking change or not. As such, we must explain what was broken and why, and what you can do about it.

As for the rest, I'd prefer changelog to contain the minimal necessary information. I wouldn't expect everyone to scourge PRs for information, unless we fail to explain it properly at which point users won't have any other option other than to go into the PR after all.

Reply:
Moeroever, the changelog should link to README — which is the authoritative documentation for the changes. Without the links, users would be left to go through the ever bigger readme by themselves, which is not good.